UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -226,9 +226,15 @@ public HungerState CalculateHungerState()
 		/// </summary>
 		private List<Sickness> immunedSickness;
 
+		public PlayerScript PlayerScriptOwner;
+
 		public virtual void Awake()
 		{
 			EnsureInit();
+			if(PlayerScriptOwner == null)
```

Comment:
you can just do 
LivingHealthMasterBase as playerhealthv2
to get this, ideally we shouldn't need to get player code at all since it would be perfectly integrated with LivingHealthMasterBase  and things that go with it